magenta/models/nsynth/wavenet/fastgen.py

```diff
@@ -161,8 +170,9 @@ def save_batch(batch_audio, batch_save_paths):
 
 def synthesize(encodings,
                save_paths,
+               FLAGS=None,
                checkpoint_path="model.ckpt-200000",
-               samples_per_save=1000):
+               samples_per_save=10000):
```

Comment:
Any reason for changing this default? 1000 was nice to reduce the amount of time to first output.

Reply:
Yes, 1000 is good, but since saving output reduces the runtime performance (throughput), I increased it to 10000. If you think, 1000 is needed, that is also fine with me.